src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -359,9 +359,7 @@ void EnBom_Draw(Actor* thisx, PlayState* play) {
     s32 pad;
     EnBom* this = (EnBom*)thisx;
 
-#ifdef OOT_DEBUG
-    if (1) {}
-#endif
+    if (OOT_DEBUG) {}
```

Comment:
this doesn't match for me

Reply:
Uhh, it does for me...

Reply:
You're right, I swear I had tested this so I'm not sure what I did wrong